plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java

```diff
@@ -190,6 +193,33 @@ public synchronized int getProgressBar() {
 
     private final Provider<UpdateManager> updateManagerProvider;
     private final Provider<BeakerObjectConverter> objectSerializerProvider;
+    private static final Pattern CONTROL_CHARACTERS_IN_STRING_PATTERN
+        = Pattern.compile("[^\\p{Cc}]*[\\p{Cc}]+[^\\p{Cc}]*(?U)");
```

Comment:
why use this complicated pattern instead of just calling isISOControl in a loop?